README.md

```diff
@@ -31,24 +31,33 @@ There is now a facility to deep search for issues matching a content regex and a
 
 ### Examples
 
+#### Generate a Jira access token
+
+1. Make sure you are connected to Red Hat VPN.
```

Comment:
AFAIK VPN is not needed to access jira, so you can get the PAT without being connected

Reply:
Done. 
I've added this line above and clarified that it is needed to download the logs